routes.go

```diff
@@ -243,10 +261,16 @@ func metricMiddleware() (out MetricMiddlewareOut) {
 			),
 			fx.Annotate(
 				func(si touchhttp.ServerInstrumenter) {
-					out.Primary = alice.New(si.Then)
+					out.Alternate = alice.New(si.Then)
 				},
 				fx.ParamTags(`name:"server_health"`),
 			),
+			fx.Annotate(
+				func(si touchhttp.ServerInstrumenter) {
+					out.Primary = alice.New(si.Then)
```

Comment:
Functions declared as part of DI should not refer to variables declared outside.  `out` should refer to a function parameter, which would be injected.  For example:

```
func(r *mux.Router, si touchttp.ServerInstrumenter) {
    r.Use(si.Then)
}
```

`arrange` exposes the `mux.Router` for exactly this purpose.

If you need to inject a named `mux.Router`, you would either (1) declare an `fx.In` struct that had the router and instrumenter, or (2) use `fx.Annotate` to annotate the function appropriately.